test/mitmproxy/test_examples.py

```diff
@@ -1,11 +1,33 @@
 import glob
+import os
+from contextlib import contextmanager
+
 from mitmproxy import utils, script
 from mitmproxy.proxy import config
-from . import tservers
+from netlib import tutils as netutils
+from netlib.http import Headers
+from . import tservers, tutils
+
+example_dir = utils.Data(__name__).path("../../examples")
+
+
+@contextmanager
+def example(command):
+    command = os.path.join(example_dir, command)
+    ctx = DummyContext()
```

Comment:
@mhils: Another issue with using the contextmanager is that I'll have to replace the DummyContext with a `script.ScriptContext()` since we'll need it's features like `add_content_view`.